src/reducers/scrollReducer.js

```diff
@@ -0,0 +1,12 @@
+import { TOGGLE_SCROLL } from '../actions/actionTypes';
+
+const INITIAL_STATE = true;
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case TOGGLE_SCROLL:
+      return !state;
+    default:
+      return state;
+  }
+};
```

Comment:
This functionality seems weird to me, example this switch statement's cases. Think about this a bit and what the case statements should be checking.

Reply:
Left this unfixed for a future dev kaizen